modules/quicksearch.php

```diff
@@ -356,7 +356,7 @@ function macformat($mac, $escape = false)
                     } else {
                         $action = '?m=customerinfo&id=' . $row['id'];
                     }
-                    $name = truncate_str($row['customername'], 50);
+                    $name = '(#' . $row['id'] . ') ' . truncate_str($row['customername'], 50);
```

Comment:
Przy wyszukwaniu kont VoIP lub numerów telefonów zawsze id klienta w sugestii? Gdy znajdzieny konto VoIP to nie powinno być jego **id** w sugestii?

Reply:
Konto VoIP jest zawsze przypisane do klienta. Czy się mylę?

Reply:
Baza: https://github.com/chilek/lms-plus/blob/master/doc/lms.pgsql#L602-L603 - brak przymusu,
Formularze add, edit - przymus. Może warto to zunifikować?

Reply:
@chilek co z tym? Czeka na Twoją decyzję.